File: sdkwork-backend-sdk-kotlin/src/main/kotlin/com/sdkwork/backend/PlusApiResultListPlusShareVisitRecordVO.kt
package com.sdkwork.backend

data class PlusApiResultListPlusShareVisitRecordVO(
    val data: List<PlusShareVisitRecordVO>? = null,
    val code: String? = null,
    val msg: String? = null,
    val requestId: String? = null,
    val ip: String? = null,
    val hostname: String? = null,
    val errorName: String? = null
)

File: sdkwork-backend-sdk-python/sdkwork_backend_sdk/models/plus_api_result_list_plus_share_visit_record_vo.py
from dataclasses import dataclass
from typing import Optional, List, Dict, Any

@dataclass
class PlusApiResultListPlusShareVisitRecordVO:
    """API调用结果"""
    data: List[PlusShareVisitRecordVO]
    code: str
    msg: str
    request_id: str
    ip: str = None
    hostname: str = None
    error_name: str
